app/services/testdata/faker/DataFaker.scala

```diff
@@ -731,35 +736,56 @@ object DataFaker {
 
     object Event {
       def id = UUIDFactory.generateUUID()
+
       def eventType = randOne(List(EventType.FSAC, EventType.TELEPHONE_INTERVIEW, EventType.SKYPE_INTERVIEW,
```

Comment:
I realise this isn't your change, but this duplication of skype interview looks a bit suspect do you think?